zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -1562,9 +1755,56 @@ components:
             description: The key of the first process instance the message correlated with
             type: integer
             format: int64
+    DocumentReference:
+      type: object
+      properties:
+        $documentType:
```

Comment:
🔧 As discussed, do we need this to be prefixed with `$`?

Reply:
Right, let's drop this 👍